modules/new_profile/php/addprofile.class.inc

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * The validateIDs endpoint is used to verify whether or not
+ * a candidate PSCID and CandID match
+ *
+ * It takes two parameters (PSCID and CandID) and returns either
+ * "1" or "0".
```

Comment:
The doc for the function is fixed, but I just noticed it's still wrong up here..